etcdmgmt/etcd-mgmt.go

```diff
@@ -35,6 +36,11 @@ func StartEtcd() error {
 		"-initial-advertise-peer-urls", initialAdvPeerUrls,
 		"--initial-cluster", "default="+listenPeerUrls)
 
+	// Don't kill chlid process (etcd) upon ^C (SIGINT) of main glusterd process
```

Comment:
You were partially right that if a SIGKILL/SIGTERM is sent to the process, child process remains as it is but thats only true for a foreground process. If glusterd2 is run in background its not correct.